armik:
    n0:
        arm_segment1_len: 15
        arm_segment2_len: 20
        rotation: "/shoulder_pan_controllerL/command"
        shoulder: "/shoulder_tilt_controllerL/command"
        shoulderi: false
        elbow: "/elbow_tilt_controllerL/command"
    n1:
        arm_segment1_len: 15
        arm_segment2_len: 20
        rotation: "/shoulder_pan_controllerR/command"
        shoulder: "/shoulder_tilt_controllerR/command"
        shoulderi: true
        elbow: "/elbow_tilt_controllerR/command"
